src/mlpack/bindings/go/print_doc.hpp

```diff
@@ -16,7 +16,7 @@
 #include <mlpack/prereqs.hpp>
 #include <mlpack/core/util/hyphenate_string.hpp>
 #include "get_go_type.hpp"
-#include "camel_case.hpp"
+#include <mlpack/bindings/utils/camel_case.hpp>
```

Comment:
So, there also exists the directory `src/mlpack/core/util/` which contains things in the namespace `mlpack::core::util`.  Maybe it is better here to match that name with `mlpack::bindings::util`?  (Instead of having `util` there and `utils` here.)

Reply:
Done.